tests/PageObjects/RemoteMic/RemoteMicMainPage.ts

```diff
@@ -50,8 +50,12 @@ export class RemoteMicMainPagePO {
     return this.page.getByTestId('remote-keyboard');
   }
 
+  public get enterKeyboardButton() {
+    return this.page.getByTestId('keyboard-enter');
+  }
+
   public async pressEnterByKeyboard() {
```

Comment:
I know it's not changed in this PR, but we could rename it to `pressEnterOnRemoteMic` to be consistent with other methods